mlir/Maintainers.md

```diff
@@ -44,6 +44,12 @@ MLIR components pertaining to egress flows from MLIR, in particular to LLVM IR.
   [@gysit](https://github.com/gysit) (GitHub),
   gysit (Discourse)
 
+### Dialects
+
+#### CPU Dialects
+
+* 'vcix' Dialect ([@mshockwave](https://github.com/mshockwave))
+
```

Comment:
Not sure how to layer this - should this be a sub-section of Egress?

Reply:
I'd rather do this in one go than having each one adding their own, at least in this initial period. Later, we can do that one by one.